graph.go

```diff
@@ -75,7 +75,7 @@ func (c *GraphCmd) Execute(args []string) error {
 
 	// Add link annotations.
 	dataLCStr := string(bytes.ToLower(data))
-	df, err := dockerfile.Decode(bytes.NewReader(data))
+	df, err := df.Decode(bytes.NewReader(data))
```

Comment:
This name shadowing would've been a problem if the (previously named) `dockerfile` package was needed below. But it's not, so this is fine as is.